### ISSUE TITLE
ISlideShapes.AddPicture() doesn't add picture

### ISSUE THREAD

**Issue by ashahabov** [ShapeCrawler/ShapeCrawler#649]
`ISlideShapes.AddPicture()` method doesn't add picture on slide.

### Client Code to Reproduce
```c#
var pres = new Presentation("picture-adding.pptx");
using var image = new FileStream("picture.png", FileMode.Open, FileAccess.Read);

pres.Slides[0].Shapes.AddPicture(image);

pres.Save();
```
[picture-adding.pptx](https://github.com/ShapeCrawler/ShapeCrawler/files/14858567/picture-adding.pptx)
[picture.png](https://github.com/ShapeCrawler/ShapeCrawler/assets/5969304/869ac077-7225-4da4-9719-92af106a7025)

### Actual Result
The slide is empty.

### Expected Result
The slide should display a picture shape containing the image from the file `picture.png`.

### Discussed in https://github.com/ShapeCrawler/ShapeCrawler/discussions/480

<div type='discussions-op-text'>

<sup>Originally posted by **Stravit
oMike** March 23, 2023</sup>
Hi, 

I'd like to add a slide with an image, is it possible to e.g. add a new slide last in the presentation and then insert an image ?</div>

**Comment by Luigicube** [comment]
I think the picture is added but either at a non-visible location or with a non-relevant size.
Changing size and location works.

**Comment by jcoliz** [comment]
Do you folks want to try this again? I cannot repro this on latest. This works just fine for me on latest master (0e5d22db468e9d32e1c37bfe4577edf7b6822b7a). It's possible that this was fixed by fixing #671.

Actually now that I think about it, this is almost surely a duplicate of #671.
